app/views/content/life-as-a-teacher/pay-and-benefits/_block-promos.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="row">
+  <section class="container">
+    <section class="image-blocks">
+      <%= render CallsToAction::MailinglistBlockComponent.new(
+      text: "Sign up for free guidance and support on getting into teaching, including learning more about the benefits of a career in teaching."
+    ) %>
+      <%= render CallsToAction::RoutesBlockComponent.new %>
+      <%= render CallsToAction::AdviserBlockComponent.new(
```

Comment:
Do we want to put advisers on the pages in this section? I'm not sure - we get a lot of casual traffic to this page!